### DISCUSSION TITLE
How to start minimized

### DISCUSSION THREAD

**Discussion by Qu1goN** [rocksdanister/lively#1005] [Q&A]
Hi, my english is not the best :)
Lively starts automatically, but not minimized. I always have to minimize it manually.
In the settings there is no or I don't see any setting to start it minimized.
So how can I get it started minimized?

**Comment by rocksdanister** [comment]
This is a bug, Lively should start minimized normally.

**Comment by Qu1goN** [comment]
Do not do it. Is that a registry entry - where can I find it? Possibly I can change it then.

**Reply by rocksdanister** [reply]
No, this is not normal and I don't know the reason without further information.

Are you perhaps using Nvidia Quadro gpu with studio drivers?

**Reply by Qu1goN** [reply]
No i use a GT 710 with 472.12-desktop-win10-win11-64bit-international-dch-whql Driver.

**Comment by Qu1goN** [comment]
I delete the registry autostatart entry in Computer\HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Run

There was this Entry: C:\Program Files (x86)\Lively Wallpaper\livelywpf.exe

And in C:\ProgramData\Microsoft\Windows\Start Menu\Programs is a shortcut an i check the drop down box and choose minimized

Now it works. thx. I will soon reinstall the system (maybe switch to win 11 right away) and hope that it will work normally again.
